src/views/inspector/InspectorViewProvider.ts

```diff
@@ -93,11 +105,9 @@ export class InspectorViewProvider implements vscode.WebviewViewProvider {
       if (data.type === 'variableOrFeature') {
         this.selectedType = data.value
         if (this.selectedType === 'Variable') {
-          const variables = Object.keys(this.variables)
-          this.selectedKey = variables[0] || ''
+          this.selectedKey = this.orderedVariables[0]?.key || ''
```

Comment:
we're getting this default value in a few places, do you think this could use a `getDefaultSelectedKey` method that accepts the type and returns the appropriate key/id?